core/commands/routing.go

```diff
@@ -254,6 +263,18 @@ Trigger reprovider to announce our data to network.
 			return ErrNotOnline
 		}
 
+		// respect global config
+		cfg, err := nd.Repo.Config()
+		if err != nil {
+			return err
+		}
+		if !cfg.Provider.Enabled.WithDefault(config.DefaultProviderEnabled) {
+			return errors.New("invalid configuration: Provider.Enabled is set to 'false'")
+		}
+		if cfg.Reprovider.Interval.WithDefault(config.DefaultReproviderInterval) == 0 {
+			return errors.New("invalid configuration: Reprovider.Interval is set to '0'")
+		}
```

Comment:
Currently, setting `Reprovider.Interval` to `0` disables periodic reprovides, however manual reprovides (triggered by the `reprovide` command) are still possible. Do we want to disable this feature?